docs/wiki/EL/JSON-RPC.md

```diff
@@ -0,0 +1,113 @@
+# JSON-RPC
+
+---
+
+The JSON-RPC specification is a remote procedure call protocol encoded in JSON based on [OpenRPC](https://open-rpc.org/getting-started). It allows calling functions on a remote server, and for the return of the results.
+It is part of the Execution API specification which provides a set of methods to interact with the Ethereum blockchain.
+It is better known to be the way of how the clients interact with the network, even how the consensus layer (CL) and the execution layer (EL) interact.
+This section provides a description of the JSON-RPC methods.
+
+## JSON-RPC Methods
+
+There are a lot of JSON-RPC methods, but for summarizing purposes here are divided into the following prefix categories: *debug*, *engine* and *eth*, and all of them share a common structure:
```

Comment:
I would highlight here there although there are more namespaces used by clients (admin, net, txpool, ots, etc), only these are specified and must behave the same across all implementations.